dev/bots/test.dart

```diff
@@ -294,7 +294,10 @@ Future<Null> _runSmokeTests() async {
 Future<Null> _runToolTests() async {
   await _runSmokeTests();
 
-  await _pubRunTest(path.join(flutterRoot, 'packages', 'flutter_tools'));
+  await _pubRunTest(
+    path.join(flutterRoot, 'packages', 'flutter_tools'),
+    enableAsserts: true,
```

Comment:
maybe call this enableFlutterToolAsserts to clarify that it doesn't affect whether pub run itself, or the tests themselves, have asserts enabled.